eng/versioning/version_client.txt

```diff
@@ -452,9 +452,9 @@ com.azure.resourcemanager:azure-resourcemanager-migrationdiscoverysap;1.0.0-beta
 com.azure.resourcemanager:azure-resourcemanager-storageactions;1.0.0-beta.1;1.0.0-beta.2
 com.azure.tools:azure-sdk-archetype;1.0.0;1.2.0-beta.1
 com.azure.tools:azure-sdk-build-tool;1.0.0;1.1.0-beta.1
-com.generic:generic-core;1.0.0-beta.1;1.0.0-beta.1
-com.generic:generic-json;1.0.0-beta.1;1.0.0-beta.1
-com.generic:generic-core-http-okhttp;1.0.0-beta.1;1.0.0-beta.1
+io.clientcore:core;1.0.0-beta.1;1.0.0-beta.1
+io.clientcore:core-json;1.0.0-beta.1;1.0.0-beta.1
+io.clientcore:core-http-okhttp;1.0.0-beta.1;1.0.0-beta.1
```

Comment:
My naming gist suggested we call the http libraries `http-okhttpX`, etc. Two differences: the missing `core-`, and the presence of a specific okhttp version (so that we can support side-by-side `http-okhttp3`, `http-okhttp4`, etc). Let's discuss pros and cons in our team chat.